README.md

```diff
@@ -510,7 +514,7 @@ Previous releases may also have been signed with one of the following GPG keys:
 ### Working Groups
 
 Information on the current Node.js Working Groups can be found in the
-[CTC repository](https://github.com/nodejs/CTC/blob/master/WORKING_GROUPS.md).
+[TSC repository](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md).
```

Comment:
This change means we'll need to merge the two WORKING_GROUPS.md docs. If anyone's feeling ambitious and wants to open a PR against the TSC repo to have that ready to go, that would be awesome. (Non-blocking, just pointing out something that someone may choose to act on at this time or not.)